### DISCUSSION TITLE
Drag 组件

### DISCUSSION THREAD

**Discussion by yangjinjun3** [jdf2e/nutui-react#12] [General]
功能、基本交互测试
1、拖动时控制台报错
...
文档完整性验证
...
...
props 所有属性测试
1、无法自定义class和style
...
props 异步修改组件是否动态更新
...
...
emits 所有事件测试
...
...
npm run build 子组件是否被构建
...
其它

**Comment by xiaoyatong** [comment]
1. 拖动时，
touchMove @ drag.tsx:94
drag.tsx:94 [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.

2. 
![image](https://user-images.githubusercontent.com/84436086/139414808-d65e16b7-c984-4616-826d-6ecb38f6b226.png)
